### PR TITLE
fix: a slight modification to test-data.xml makes the tests fail (DEV-1140)

### PR DESCRIPTION
**Pull request by jnussbaum**
resolves DEV-1140

### CODE REVIEW

File: knora/dsplib/utils/xml_upload.py [hunk 1]
```diff
@@ -321,6 +321,7 @@ def xml_upload(input_file: str, server: str, user: str, password: str, imgdir: s
     if len(stashed_xml_texts) > 0:
         try:
             nonapplied_xml_texts = _upload_stashed_xml_texts(verbose, id2iri_mapping, con, stashed_xml_texts)
+            nonapplied_xml_texts = _purge_stashed_xml_texts(nonapplied_xml_texts, id2iri_mapping)
```
Comment (irinaschubert) [inline knora/dsplib/utils/xml_upload.py:324]: Are you sure you want to call these variables the same?

File: knora/dsplib/utils/xml_upload.py [hunk 2]
```diff
@@ -329,6 +330,7 @@ def xml_upload(input_file: str, server: str, user: str, password: str, imgdir: s
     if len(stashed_resptr_props) > 0:
         try:
             nonapplied_resptr_props = _upload_stashed_resptr_props(verbose, id2iri_mapping, con, stashed_resptr_props)
+            nonapplied_resptr_props = _purge_stashed_resptr_props(stashed_resptr_props, id2iri_mapping)
```
Comment (irinaschubert) [inline knora/dsplib/utils/xml_upload.py:333]: same as above

**Review by irinaschubert — APPROVED** [review]
I don't understand the changes but I assume you have tested them and they work?

**Review by BalduinLandolt — APPROVED** [review]
I don't have much to contribute to that one

**Pull request merged**